### DISCUSSION TITLE
Deconstructed clothes should yield organic fiber or bandages.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5925] [Ideas and suggestions]
Deconstructed uniforms of any type should yield either 1-2 organic fiber or 2-4 bandages. Obtaining organic fiber this way may be OP, however, I think bandages should be fine, given that you need a lot to heal severe injuries.

Barotrauma is a game about resourcefulness and brutality. What could be more brutal than being forced to remove the clothes from your fallen comrades for medical supplies?

**Comment by ViktorShield** [comment]
I don't really think it's a good idea, since any spawned player already has a uniform. It's some sort of dupe. I personally believe that the best way would be not giving any equipment on new player spawn, if it's not on the very start of the campaign itself. 

So, every time a players disconnect, they leave a corpse, containing items for others for later use. There would be a few extra containers filled with corresponding armor, for example. 

For example, I am a medic, who played the very first round of the campaign. If I leave, I either drop all my items in a corresponding cabinet, or let others loot me. Another player, joining the campaign AFTER it starts, will have to use other's clothes. 

This way we could make it "more brutal" and maybe logical. Hosts(or moderators) will from now on will have to buy clothes on the outposts(esp. habitation and city outposts), or loot others for supplies. Maybe even take clothes from corpses at the wreckage

This only applies to campaign with a specific flag in settings. 

If so, clothes could really be recycled for 0.5 units of organic fiber, or crafted with 1 or 0.5 of organic fiber and a corresponding paint. Enginner's clothes, for example, may contain 0.25 units of lead(since engineer's clothes protect from radiation) and some yellow paint.  Security clothes - 0,25 bulletproof fiber and so on.
